packages/server/package.json

```diff
@@ -115,6 +115,7 @@
     "trash": "5.2.0",
     "tree-kill": "1.2.2",
     "ts-node": "8.5.4",
+    "tslib": "^2.3.0",
```

Comment:
Might need to re-yarn so the lockfile gets touched in order for this to get picked up, I'm guessing that the circle cache of yarn install is what's still causing the failures.